R/test-that.R

```diff
@@ -38,30 +38,26 @@ test_code <- function(test, code, env = test_env(), skip_on_empty = TRUE) {
   }
 
   ok <- TRUE
-  register_expectation <- function(e) {
-    calls <- e$expectation_calls
+  register_expectation <- function(e, debug_end) {
```

Comment:
Could you add a comment explaining what `debug_end` is?

Reply:
:white_check_mark: